control-plane/pkg/apis/sources/v1beta1/kafka_defaults.go

```diff
@@ -71,4 +71,5 @@ func (k *KafkaSource) SetDefaults(ctx context.Context) {
 	}
 
 	k.Spec.Sink.SetDefaults(ctx)
-}
+	k.Spec.Delivery.SetDefaults(ctx)
+}
```

Comment:
Linter complains on missing newline at the end of the file

https://github.com/knative-sandbox/eventing-kafka-broker/actions/runs/4412829966/jobs/7741713539

```suggestion
}

```

Reply:
Fixed that. That's because I did dummy commit to restart CLA check.